boards/arm/nucleo_f446ze/nucleo_f446ze.dts

```diff
@@ -0,0 +1,214 @@
+/*
+ * Copyright (c) 2018 Philémon Jaermann
+ * Copyleft (¬c) 2021 Tom Owen
```

Comment:
To my best knowledge, we don't have a copyleft policy in Zephyr, this can't be merged as is.
@cfriedt any idea on how to proceed here ?

Reply:
I've never seen this before. However, we definitely require a copy-right (name + year) as well as an SPDX license id